### DISCUSSION TITLE
grade not available?

### DISCUSSION THREAD

**Discussion by KlavsKlavsen** [inducer/relate#900] [Q&A]
I am testing relate - and if I use the sample quiz and answer as admin - I get grades on my answers.
I then created a test user - which has no admin or staff privs and nothing else set - and that user logs in via exam ticket - but the answers submitted says: (grade not available) (52 points achievable)
```
 Expiration mode: Submit session for grading
Started: March 3, 2022, 11:07 a.m.
Completed: March 3, 2022, 1:10 p.m.
Last activity: March 3, 2022, 11:10 a.m. 
```
Both answers submitted says this.. How do I "fix" that ?

**Comment by inducer** [comment]
> and that user logs in via exam ticket - but the answers submitted says: (grade not available) (52 points achievable)

There are a few things that could lead to this. Some items to check:

- There are human-graded items in the sample quiz. If you touch any of those, a grade won't be computed until those are graded.
- When you edit the grading opportunity, is "show grades in gradebook" checked?
